backend/contributions/migrations/0084_enable_builder_review_hierarchy.py

```
@@ -0,0 +1,31 @@
+from django.db import migrations
+
+
+def enable_builder_review_hierarchy(apps, schema_editor):
```

Ruff (0.15.21)

[warning] Unused function argument: `schema_editor`

(ARG001)